spec/helpers/lib/resourceHints.js

```diff
@@ -0,0 +1,151 @@
+const _ = require('lodash');
+const Code = require('code'),
+    expect = Code.expect;
+const Sinon = require('sinon');
+const Lab = require('lab'),
+    lab = exports.lab = Lab.script(),
+    describe = lab.experiment,
+    it = lab.it,
+    beforeEach = lab.beforeEach,
+    afterEach = lab.afterEach;
+const {
+    addResourceHint,
+    defaultResourceHintSate,
+    resourceHintAllowedTypes
+} = require('../../../helpers/lib/resourceHints');
+
+describe('resource hints', function () {
+
+    describe('addResourceHint', function () {
+
+        let sandbox;
+
+        beforeEach(done => {
+            sandbox = Sinon.createSandbox();
+            done();
+        });
+
+        afterEach(done => {
+            sandbox.restore();
+            done();
+        });
+
+        it("does work as expected with valid params", (done) => {
```

Comment:
Could you please name the test as what it does? It's not really clear from the test name what is it doing or what is it behaviour